### PR TITLE
fix(server: io: rfc822): correct handling of field 'from'

### PR DESCRIPTION
**Pull request by actionless**

### CODE REVIEW

**Comment by actionless** [comment]
to run this test:
`docker-compose -p vpps -f docker/docker-compose.yml run backend ./scripts/fig_wrapper.sh nosetests -sv --logging-level ERROR -m rfc`

File: server/superdesk/io/rfc822_parser_test.py [hunk 1]
```diff
@@ -35,6 +46,11 @@ def test_headline(self):
     def test_body(self):
         self.assertEquals(self.items[0]['body_html'].strip(), '<div>body text<br/><div>\n</div></div>')
 
+    def test_rfc_from(self):
+        self.assertEqual(self.items[0]['original_source'],
+                         'a sender <asender@a.com.au>')
+        self.assertEqual(self.items[0]['original_creator'], self.user_id)
```
Comment (actionless) [inline server/superdesk/io/rfc822_parser_test.py:52]: or mb add one more test for handling unknown mail?

**Comment by thnkloud9** [comment]
:thumbsup:

File: server/superdesk/io/rfc822_parser_test.py (hunk 1 above)
Comment (thnkloud9) [inline server/superdesk/io/rfc822_parser_test.py:52]: actually, yeah, probably a good idea.

File: server/superdesk/io/rfc822.py [hunk 2]
```diff
@@ -23,11 +24,25 @@
...
 
 logger = logging.getLogger(__name__)
 
 
+class UserNotRegisteredException(Exception):
+    pass
+
+
+def get_user_by_email(from_field):
+    email_address = re.compile('^.*<(.*)>$').findall(from_field)[0]
```
Comment (actionless) [inline server/superdesk/io/rfc822.py]: oops, i was going to move the compilation of the regex outside of the function, will fix it tomorrow

**Pull request closed without merging**